x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/utils/ingest/PipelineRegistry.java

```diff
@@ -85,9 +86,18 @@ protected boolean isClusterReady(ClusterChangedEvent event) {
             return false;
         }
 
+        if (getMinClusterVersion() != null && event.state().nodes().getMinNodeVersion().onOrBefore(getMinClusterVersion())) {
```

Comment:
We do want this to happen in 8.8.0, right? Shouldn't this read like:

```suggestion
        if (getMinClusterVersion() != null && event.state().nodes().getMinNodeVersion().before(getMinClusterVersion())) {
```

Reply:
I find it easier to read with:

```
Version minNodeVersion = event.state().nodes().getMinNodeVersion();
if (getMinClusterVersion().after(minNodeVersion) {
  return false;
}
...
```